### ISSUE TITLE
About the Decomposition Table

### ISSUE THREAD

**Issue by nanmumc** [GTNewHorizons/NewHorizons#1107]
![5qyyb0b_6 _nqy6qsbw7 v](https://cloud.githubusercontent.com/assets/25528572/22877391/34508876-f210-11e6-8dff-ddab24af350f.png)
![tl15ygl0uqi j9 3je w m](https://cloud.githubusercontent.com/assets/25528572/22877392/363c5674-f210-11e6-9b7c-20742c09e42d.png)
![cg spzeeny2faor 4p oua](https://cloud.githubusercontent.com/assets/25528572/22877397/385ee156-f210-11e6-99f3-d96d9a980af3.png)

Because the ID of Wood Plank is same to The holy Planks of Sengir,the Decomposition Table can't distinguish the NBT.

![e9ic43d nw pic0y eq2o07](https://cloud.githubusercontent.com/assets/25528572/22877417/4898788e-f210-11e6-96f1-82f9f05bd575.png)

**Comment by Dream-Master** [comment]
decompositionBlacklist	text	(empty)

minecraft:diamond_sword - Diamond Sword of any damage value
minecraft:stonebrick/3 - Chiseled Stone Brick
minecraft:dye/1+11 - Red and Yellow Dyes

Special cases:

dispenser - Dispenser - vanilla blocks and items do not need the minecraft: prefix
minecraft:* - All vanilla blocks & items - * is a wildcard that blacklists entire mod

**Comment by Ugachaga** [comment]
https://github.com/chylex/Hardcore-Ender-Expansion/issues/145 & [InfiTech config](https://github.com/JasonMcRay/InfiTech-1.7/blob/a9b80e41b90c2a7488dac4fbb0369171ef3d91b4/config/HardcoreEnderExpansion.cfg#L20-L92)

**Comment by Dream-Master** [comment]
Yes but disabled the table are better. The tf table I disabled too.
